src/parser/jobs/mnk/Fists.js

```diff
@@ -58,15 +107,105 @@ export default class Fists extends Module {
 			content: <Fragment>
 				Fist buffs are one of your biggest DPS contributors, either directly with <ActionLink {...ACTIONS.FISTS_OF_FIRE} /> or <StatusLink {...STATUSES.GREASED_LIGHTNING_I} /> manipulation with <ActionLink {...ACTIONS.FISTS_OF_EARTH} /> and <ActionLink {...ACTIONS.FISTS_OF_WIND} />.
 			</Fragment>,
-			why: `No Fist buff was active for ${this.parser.formatDuration(this._stanceless)}.`,
+			why: `No Fist buff was active for ${this._fistGCDs[STANCELESS]} GCDs.`,
```

Comment:
This is a bit nitpicky but, why not something like `${this._fistGCDs[STANCELESS]} GCDs had no fist buff active.` Or could use the plural i18n -- When you implement it.

Reply:
Sure, this is a pretty old module and better wording is always good. I intend to implement i18n on the weekend but worth fixing now.